src/test/unit/alignsensor_unittest.cc

```diff
@@ -101,70 +99,70 @@ static void initZAxisRotation(int32_t mat[][3], int32_t angle)
 
 #define TOL 1e-5 // TOLERANCE
 
-static void alignSensorViaMatrixFromRotation(float *dest, sensor_align_e alignment)
+static void alignSensorViaMatrixFromRotation(vector3_t *dest, sensor_align_e alignment)
 {
-    fp_rotationMatrix_t sensorRotationMatrix;
+    matrix33_t sensorRotationMatrix;
 
     sensorAlignment_t sensorAlignment;
 
     buildAlignmentFromStandardAlignment(&sensorAlignment, alignment);
 
-    buildRotationMatrixFromAlignment(&sensorAlignment, &sensorRotationMatrix);
+    buildRotationMatrixFromAngles(&sensorRotationMatrix, &sensorAlignment);
 
     alignSensorViaMatrix(dest, &sensorRotationMatrix);
 }
 
 static void testCW(sensor_align_e rotation, int32_t angle)
 {
-    float src[XYZ_AXIS_COUNT];
-    float test[XYZ_AXIS_COUNT];
+    vector3_t src;
```

Comment:
```suggestion
    { 
        vector3_t src = {.x=1, .y=0, .z=0};
```

or maybe constructor-like function: `vector3_t vector3(float x, float y, float z)`